src/L2GovernorMetadata.sol

```diff
@@ -28,7 +38,21 @@ abstract contract L2GovernorMetadata {
     virtual
   {
     _proposals[proposalId] = Proposal(voteStart, voteEnd, isCanceled);
-    emit ProposalAdded(proposalId, voteStart, voteEnd, isCanceled);
+    if (isCanceled) {
+      emit ProposalCanceled(proposalId);
+    } else {
+      emit ProposalCreated(
+        proposalId,
+        address(0),
+        new address[](0),
+        new uint256[](0),
+        new string[](0),
+        new bytes[](0),
```

Comment:
Fine with the empty params here, as we don't have any better info. I do wonder whether it will cause any issues with Tally, but we'll have to test that out.